src/schematools/types.py

```diff
@@ -527,14 +498,31 @@ def use_dimension_fields(self) -> bool:
         return self._parent_schema.use_dimension_fields
 
     @property
-    def temporal(self) -> Optional[Dict[str, Union[str, Dict[str, List[str]]]]]:
-        """Return the temporal info from the dataset schema """
-        return self._parent_schema.fetch_temporal(field_modifier=lambda x: x)
+    def temporal(
+        self, field_modifier: Optional[Callable] = None
+    ) -> Optional[Dict[str, Union[str, Dict[str, List[str]]]]]:
+        """Return the temporal info"""
```

Comment:
You might want to document what `field_modifier` parameter is used for/actually does.

Reply:
Nitpick: never call something "info" or "information", because it's meaningless. Everything in the computer memory is data or information. Better describe something like `"""Provides the configuration of the temporal table"""` or `"""Defines whether the table is temporal, and which fields are used to identify historical records."""`.